website/content/docs/getting-started.md

```diff
@@ -0,0 +1,163 @@
++++

```

Comment:
This is html because it's copy pasted from the site example to have some meat in the docs until we write everything down, but note that this is a markdown file and hugo can process everything.